docs/cli.md

```diff
@@ -1218,39 +1291,59 @@ For more information, see [Train a Model with a Custom Python Training Script](/
+| `--visibility` | Visibility of the registry item, can be `public`, `private`, or `draft`. | `update` | **Required** |
 | `--version` | Version of the ML training script to upload. | `upload` | Optional |
+| `--description` | Description of the ML training script. | `update` | Optional |
 | `--framework` | Framework of the ML training script to upload, can be `tflite`, `tensorflow`, `pytorch`, or `onnx`. | `upload` | Optional |
+| `--url` | URL of GitHub repository associated with the training script. | `upload` | Optional |
 | `--type` | Task type of the ML training script to upload, can be `single_label_classification`, `multi_label_classification`, or `object_detection`. | `upload` | Optional |
 | `--draft` | Indicate draft mode, drafts are not viewable in the registry. | `upload` | Optional |
-| `--visibility` | Visibility of the registry item, can be `public`, `private`, or `draft`. | `update` | **Required** |
-| `--description` | Description of the ML training script. | `update` | Optional |
 
 ### `train`
 
 Use a training script to train an ML model on data.
 
+```sh {class="command-line" data-prompt="$"}
+viam train submit managed --dataset-id=<dataset-id> --model-org-id=<model-org-id> --model-name=<model-name> --model-type=<model-type> --model-labels=<model-labels> [...named args]
+viam train submit custom from-registry --dataset-id=<dataset-id> --org-id=<org-id> --model-name=<model-name> --script-name=<script-name> --version=<version> [...named args]
+viam train submit custom with-upload --dataset-id=<dataset-id> --org-id=<org-id> --model-name=<model-name> --path=<path> --script-name=<script-name> [...named args]
+```
+
+Examples:
+
+```sh {class="command-line" data-prompt="$"}
+# submit training job on data in Viam cloud with a Viam-managed training script
+viam train submit managed --dataset-id=456 --model-org-id=123 --model-name=MyCoolClassifier --model-type=single_label_classification --model-labels=1,2,3
+
+# submit custom training job with an existing training script in the Registry on data in Viam cloud
+viam train submit custom from-registry --dataset-id=<INSERT DATASET ID> --org-id=<INSERT ORG ID> --model-name="MyRegistryModel" --model-version="2" --version="1" --script-name="mycompany:MyCustomTrainingScript"
+
+# submit custom training job with an uploaded training script on data in Viam cloud
+viam train submit custom with-upload --dataset-id=<INSERT DATASET ID> --model-org-id=<INSERT ORG ID> --model-name="MyRegistryModel" --model-type="single_label_classification" --model-version="2" --version="1" --path=<path-to-tar.gz> --script-name="mycompany:MyCustomTrainingScript"
+```
+
 #### Command options
 
 <!-- prettier-ignore -->
 | Command option | Description | Positional arguments |
 | -------------- | ----------- | -------------------- |
-| `submit` | Submits training job on data in the Viam cloud | `managed`, `custom` |
+| `submit` | Submits training job on data in the Viam cloud. | `managed`, `custom` |
```

Comment:
get, cancel, list are also options

Reply:
added! sorry I just didn''t get to that yet